Mathlib/Data/Polynomial/Div.lean

```diff
@@ -696,6 +701,11 @@ theorem eval_divByMonic_pow_rootMultiplicity_ne_zero {p : R[X]} (a : R) (hp : p
       (Nat.lt_succ_self _) (dvd_of_mul_right_eq _ this)
 #align polynomial.eval_div_by_monic_pow_root_multiplicity_ne_zero Polynomial.eval_divByMonic_pow_rootMultiplicity_ne_zero
 
+@[simp]
+lemma mul_left_modByMonic (hq : q.Monic) : (p * q) %ₘ q = 0 := by
+  rw [modByMonic_eq_zero_iff_dvd hq]
+  exact dvd_mul_left q p
```

Comment:
I was confused why this was so far from the other one, but it looks like it needs commutativity. Would it be helpful to add documentation comments to these two lemmas pointing at each other?

Reply:
Good point!